### DISCUSSION TITLE
Proposal: Creating a CLI to initialise epic-stack

### DISCUSSION THREAD

**Discussion by akash-joshi** [epicweb-dev/epic-stack#50] [Ideas]
Epic-stack is amazing. What will help it reach a wider audience is having a scaffolding CLI which helps users get started faster.

Enter `create-epic-stack`. People would be able to run `npm create epic-stack` or `yarn create epic-stack` to get started locally.

For reference, I can use [`create-t3-app`](https://github.com/t3-oss/create-t3-app/tree/next/cli). Let me know if I can try this out!

**Comment by kentcdodds** [comment]
I've definitely had the same thought and I just reserved the npm package for this: https://www.npmjs.com/package/create-epic-stack

I'm not available to work on this currently, but if someone else wants to start working on it themselves then I'd love to review it :)

**Reply by sergiodxa** [reply]
The CLI could just run `npx create-remix@latest --typescript --install --template epicweb-dev/epic-stack` for you, right? So you do `npx create-epic-stack` and it runs the other command.
